src/views/home/components/token-info-table/token-info-table.view.tsx

```diff
@@ -10,13 +10,13 @@ import { useEnvContext } from "src/contexts/env.context";
 import { ReactComponent as CopyIcon } from "src/assets/icons/copy.svg";
 import { ReactComponent as NewWindowIcon } from "src/assets/icons/new-window.svg";
 
-interface TokenDetailsProps {
+interface TokenInfoTableProps {
   token: Token;
   className?: string;
 }
 
-const TokenDetails: FC<TokenDetailsProps> = ({ token, className }) => {
-  const classes = useTokenDetailsStyles();
+const TokenInfoTable: FC<TokenInfoTableProps> = ({ token, className }) => {
```

Comment:
Technically this is not a table. I prefer naming this component `TokenDetails` or `TokenInfo`.

Reply:
We already have a `TokenInfo` component which is basically the container of this one, so renaming this to `TokenDetails` or `TokenInfo` would not help too much...